### PR TITLE
get command by input text and run to get response

### PR DESCRIPTION
**Pull request by LittleYenMin**
Needs to consider the multiple intents.

for example:
input text: `我想要查成績和曠課`
return `<Set>[昌昌;成績, 昌昌;曠課]`

### CODE REVIEW

File: .idea/jieba-bot.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dannypsnl) [inline .idea/jieba-bot.iml]: Why `.idea` do not be ignored?

File: nlu.py [hunk 2]
```diff
@@ -1,5 +1,28 @@
 import jieba
 
+command_mapping = {'曠課': '昌昌;曠課', '課表': '昌昌;課表', '成績': '昌昌;成績'}
```
Comment (dannypsnl) [inline nlu.py]: `command_map` as it's a static value

File: nlu.py [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+
 def response_message(text: str) -> str:
-    split_text = jieba.cut(text)
-    return ", ".join(split_text)
+    cmds = _get_commands(jieba.cut(text))
+    return _exec_command(cmds)
+
+
+def _get_commands(text_sequence: [str]) -> set:
+    commands = set()
```
Comment (dannypsnl) [inline nlu.py]: How about `possible_commands`?

File: nlu.py [hunk 4]
```diff
@@ -1,5 +1,28 @@
...
+    for text in text_sequence:
+        if text in command_mapping:
+            commands.add(command_mapping.get(text))
```
Comment (dannypsnl) [inline nlu.py]: How about
```py
cmd = command_map.get(text)
# rely on `get` is `get(key, default=None)` this behavior,
# NOTE: don't add the comment into code
if cmd != None:
  possible_commands.add(cmd)
```

File: nlu.py [hunk 5]
```diff
@@ -1,5 +1,28 @@
...
+    return commands
+
+
+def _exec_command(cmds: set) -> str:
```
Comment (dannypsnl) [inline nlu.py]: If you send a `set` into the function, then it should be named `_exec_commands`.

For me, why not just keep original behavior, but iterate the `set` to get all commands' result.

Then later we could have a chance to decide how to format the result into the response(grouping it or multiple-response)

File: nlu.py (hunk 4 above)
Comment (LittleYenMin) [reply nlu.py]: you're right, it more readable.

File: nlu.py (hunk 5 above)
Comment (LittleYenMin) [reply nlu.py]: You're right, iterate the set it more readable. Let's keep simple and stupid.

**Pull request merged**